### PR TITLE
Better template

### PR DESCRIPTION
**Pull request by josmo**
Adding common notification template and templating style.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ import (
 	"os"
 )
 
+const DEFAULT_TEMPLATE = "*{{build.status}}* <{{build.link}}|{{repo.owner}}/{{repo.name}}#{{truncate build.commit 8}} ({{build.branch}}) by {{build.author}}"
```
Comment (tboerger) [inline main.go]: Vet will complain about it, write DefaultTemplate instead

**Review by tboerger — CHANGES_REQUESTED** [review]
Also requires update of vendoring

File: main.go (hunk 1 above)
Comment (josmo) [reply main.go]: Should it be DefaultTemplate or defaultTemplate?  I seem to see both but I'm not sure which convention is where.

**Comment by josmo** [comment]
yep saw the vendoring so it's added.  I also made it defaultTemplate since that seems consistent with the drone code.  Let me know if it should be otherwise.

**Review by tboerger — APPROVED** [review]

**Pull request merged**